src/Data/CallbackRequest/LocalStorage/localStorageCallbackRequestDataClass.ts

```diff
@@ -1,8 +1,16 @@
+import { currentLanguage, load } from 'scrivito'
 import { provideLocalStorageDataClass } from '../../../utils/provideLocalStorageDataClass'
-import { DataClassAttributes } from '../../types'
+import { DataClassSchema } from '../../types'
 
-export function localStorageCallbackRequestDataClass(
-  attributes: DataClassAttributes,
-) {
+async function attributes(): Promise<DataClassSchema> {
```

Comment:
* [ ] FYI the function is not exported, so we don't necessarily need a return type definition. If we `return {...} as const`, we can get rid of the `import { DataClassSchema }`.

Reply:
I see. Yes sometimes TS errors are a mess. Actually the last line of the convoluted report contains the important part here. 😆